css/style.css

```diff
@@ -0,0 +1,48 @@
+body{
+    margin: 0;
+    background-color: rgba(234, 225, 201, 1);
+    font-family: "Jaini Purva";
+}
+.jaini-purva-regular {
+    font-family: "Jaini Purva", system-ui;
```

Comment:
use a full font stack. This seams unnecessary to have as a class because this is the default font set for body. Consider moving font-weight and style to body.